components/molecule/notification/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@s-ui/react-molecule-notification",
-  "version": "1.12.0",
+  "version": "1.13.0",
```

Comment:
use a bug fix instead: `1.12.1`

Reply:
Totally true. I'm sorry, it's already published with this version 😫

Reply:
The commits I made were `fix` and `refactor`. I understand that the refactor has uploaded version?